File: source/back-end/src/main/java/be/pxl/backend/service/IBordersPerProductService.java
package be.pxl.backend.service;

import java.util.List;
import be.pxl.backend.entity.BorderPerProduct;
import be.pxl.backend.entity.BorderPerProductPK;

public interface IBordersPerProductService {

	public BorderPerProduct find(BorderPerProductPK id);
	
	public List<BorderPerProduct> all();
	
	public void persist(BorderPerProduct borderPerProduct);
	
	public void delete(BorderPerProductPK id);
	
	public void update(BorderPerProduct borderPerProduct);
	
}

File: Source/frontend/frontend.Service/IBordersPerProductService.cs
﻿using frontend.Domain;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace frontend.Service
{
    public interface IBordersPerProductService
    {
        List<BorderPerProduct> All();
        BorderPerProduct Find(int id);
        void Add(BorderPerProduct bordersPerProduct);
        void Update(BorderPerProduct bordersPerProduct);
        void Delete(int id);
    }
}
